mkdocs/docs/HPC/jupyter.md

```diff
@@ -0,0 +1,129 @@
+whatis("Extensions: beniget-0.4.1, Bottleneck-1.3.7, deap-1.4.1, gast-0.5.4, mpmath-1.3.0, numexpr-2.8.7, numpy-1.26.2, pandas-2.1.3, ply-3.11, pythran-0.14.0, scipy-1.11.4, tzdata-2023.3, versioneer-0.29")
+conflict("SciPy-bundle")
+load("gfbf/2023b")
+load("Python/3.11.5-GCCcore-13.2.0")
+load("Python-bundle-PyPI/2023.10-GCCcore-13.2.0")
+load("pybind11/2.11.1-GCCcore-13.2.0")
+prepend_path("CMAKE_PREFIX_PATH","/apps/gent/RHEL8/zen2-ib/software/SciPy-bundle/2023.11-gfbf-2023b")
+prepend_path("LIBRARY_PATH","/apps/gent/RHEL8/zen2-ib/software/SciPy-bundle/2023.11-gfbf-2023b/lib")
+prepend_path("PATH","/apps/gent/RHEL8/zen2-ib/software/SciPy-bundle/2023.11-gfbf-2023b/bin")
+setenv("EBROOTSCIPYMINBUNDLE","/apps/gent/RHEL8/zen2-ib/software/SciPy-bundle/2023.11-gfbf-2023b")
+setenv("EBVERSIONSCIPYMINBUNDLE","2023.11")
+setenv("EBDEVELSCIPYMINBUNDLE","/apps/gent/RHEL8/zen2-ib/software/SciPy-bundle/2023.11-gfbf-2023b/easybuild/SciPy-bundle-2023.11-gfbf-2023b-easybuild-devel")
+prepend_path("PYTHONPATH","/apps/gent/RHEL8/zen2-ib/software/SciPy-bundle/2023.11-gfbf-2023b/lib/python3.11/site-packages")
+prepend_path("CPATH","/apps/gent/RHEL8/zen2-ib/software/SciPy-bundle/2023.11-gfbf-2023b/lib/python3.11/site-packages/numpy/core/include")
+prepend_path("LD_LIBRARY_PATH","/apps/gent/RHEL8/zen2-ib/software/SciPy-bundle/2023.11-gfbf-2023b/lib/python3.11/site-packages/numpy/core/lib")
+prepend_path("LIBRARY_PATH","/apps/gent/RHEL8/zen2-ib/software/SciPy-bundle/2023.11-gfbf-2023b/lib/python3.11/site-packages/numpy/core/lib")
+setenv("EBEXTSLISTSCIPYMINBUNDLE","numpy-1.26.2,ply-3.11,gast-0.5.4,beniget-0.4.1,pythran-0.14.0,versioneer-0.29,scipy-1.11.4,numexpr-2.8.7,Bottleneck-1.3.7,tzdata-2023.3,pandas-2.1.3,mpmath-1.3.0,deap-1.4.1")
+```
+This prints a lot of information about the module, including all libraries that are part of it. It also shows which toolchain is used in the line `load("Python/3.11.5-GCCcore-13.2.0")`.
+
+Because this is the same toolchain as the one used by the Jupyter notebook version in this example, we can include this module without errors. To do this, add the line `module load "module_name"` in the `Custom code`-field:
+
+<center>
+![image](img/ood_jupyter_custom_code.png)
+</center>
+
+If you want to check the compatibility of your Jupyter notebook version and modules without launching a notebook, you can load the module of your preferred Jupyter notebook version followed by all the modules for your libraries in a shell environment. If this does not throw any errors, the modules are compatible. For our example, this would look as follows:
+
+```shell
+$ module load JupyterNotebook/7.2.0-GCCcore-13.2.0
+$ module load SciPy-bundle/2023.07-gfbf-2023a # we expect errors since this is a module that uses a different toolchain
+Lmod has detected the following error:  ...
+
+$ module load SciPy-bundle/2023.11-gfbf-2023b # Now no errors are thrown since this module uses the same toolchain as the notebook
+```
+
+### Launching the notebook
```

Comment:
This should be the first part: standard use case, just start the notebook. Then handle "advanced use case" of also making additional Python packages available (only via available modules for now, we can add another small section on using a Python virtual env later once #639 is merged)